tests/fuzz/README.md

```diff
@@ -18,14 +18,63 @@ To run fuzzing you will need [LLVM](https://apt.llvm.org/) components:
 - lld (linker)
 - libc++
 
-## Reproducing Failure Found by Fuzzing
 
-1. Build `fuzz` test target:
+## Building fuzz tests
+
+1. Build openvino
+
+Build openvino with options `ENABLE_FUZZING` and `ENABLE_SANITIZER` enabled. It
+is recommended to use clang compiler.
+
+```bash
+(\
+mkdir -p build && cd build && \
+CC=clang-10 CXX=clang++-10 cmake .. -DENABLE_FUZZING=ON -DENABLE_SANITIZER=ON -DTREAT_WARNING_AS_ERROR=OFF && \
+cmake --build . \
+)
+```
+
+2. Build fuzz tests
+
+Build fuzz tests with options `ENABLE_FUZZING` and `ENABLE_SANITIZER` enabled.
+You should use the same compiler as was used for the openvino build.
+
 ```bash
-cmake -DENABLE_TESTS=ON .. && ninja fuzz
+(\
+mkdir -p tests/fuzz/build && cd tests/fuzz/build && \
+CC=clang-10 CXX=clang++-10 cmake .. -DENABLE_FUZZING=ON -DENABLE_SANITIZER=ON -DTREAT_WARNING_AS_ERROR=OFF -DInferenceEngine_DIR=$(pwd)/../../../build && \
+cmake --build . \
+)
 ```
 
-2. Run fuzzing test passing a failure reproducer as a command-line argument:
-``` bash
-./read_network-fuzzer crash-reproducer
+## Running fuzz tests
+
+1. Prepare fuzzing corpus
+
+Fuzzing engine needs a set of valid inputs to start fuzzing from. Those files
+are called a fuzzing corpus. Use tests/fuzz/scripts/init_corpus.py script to
+prepare fuzzing corpus.
+
+```bash
+tests/fuzz/scripts/init_corpus.py ./pdpd_layer_models/**/*.pdmodel --join pdiparams && \
+mkdir -p corpus && find ./pdpd_layer_models/ -name "*.fuzz" -exec cp \{\} ./import_pdpd-corpus \;
```

Comment:
should it be 'mkdir -p import_pdpd_corpus'?

Reply:
good catch!

Reply:
fixed